<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadmeTab</class>
 <widget class="QWidget" name="ReadmeTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QGroupBox" name="controls_group">
     <property name="title">
      <string>README Template</string>
     </property>
     <layout class="QHBoxLayout" name="controls_layout">
      <item>
       <widget class="QLabel">
        <property name="text">
         <string>Template:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="template_selector"/>
      </item>
      <item>
       <widget class="QPushButton" name="regenerate_readme_btn">
        <property name="text">
         <string>Regenerate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="editor_frame">
      <layout class="QVBoxLayout" name="editor_layout">
       <item>
        <widget class="QLabel">
         <property name="text">
          <string>README.md Content:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="readme_editor"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="preview_frame">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="preview_layout">
       <item>
        <widget class="QLabel">
         <property name="text">
          <string>Preview:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="readme_preview">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
